src/databricksbundle/notebook/GithubLinkGenerator.py

```diff
@@ -0,0 +1,89 @@
+import tomlkit
+import traceback
+
+from inspect import getsource
+from pathlib import Path
+
+
+class GithubLinkGenerator:
+    def __get_repo_name(self, module):
+        special_repos = {"daipecore": "daipe-core", "featurestorebundle": "feature-store-bundle"}
+
+        if module in special_repos.keys():
+            return special_repos[module]
+
+        return self.__get_repo_name_with_bundle(module)
+
+    def __get_repo_name_with_bundle(self, module):
+        index_of_bundle = module.find("bundle")
+
+        return module[:index_of_bundle] + "-" + module[index_of_bundle:]
+
+    def __get_module_version(self, module_name):
+        base_path = Path("/".join(list(Path.cwd().parts[0:5])))
```

Comment:
This will only work in a Databricks Repo, it's not backwards compatible with Workspace